### ISSUE TITLE
Dropdown: In PrimeNG 16.7.1  interacting with dropdown causes panel to scroll

### ISSUE THREAD

**Issue by rosenthalj** [primefaces/primeng#14039]
### Describe the bug

If a horizontal scroll bar is present, when a user clicks on the dropdown component or uses the mouse key to navigate the dropdown menu options, the panel containing the dropdown will scroll to the beginning of the visible region.

This is a similar to issue #13390 that was fixed  on September 26th.

The first video demonstrates the problem when the user clicks on the dropdown to open and then clicks on the dropdown's  "down carrot icon" to close.  

https://github.com/primefaces/primeng/assets/45439491/ca21c12b-9261-4ddf-846b-4ec7d0d95db3

![Angular_Dropdown_Component](https://github.com/primefaces/primeng/assets/45439491/9518057a-2be5-489f-a570-595efd977eb0)

The second video demonstrates the scrolling problem when the user navigates the dropdown  menu options using only the keyboard .

https://github.com/primefaces/primeng/assets/45439491/05af1ee3-62dd-424b-9f48-35b0d0a91c48

### Environment

mac os and windows os

### Reproducer

https://stackblitz.com/edit/nkd8th

### Angular version

16.x

### PrimeNG version

16.7.1

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.18.1

### Browser(s)

safari, chrome, firefox

### Steps to reproduce the behavior

To reproduce the mouse click problem
1. Run the reproducer: https://stackblitz.com/edit/nkd8th
2. Mouse Click on dropdown
3. The dropdown option menu should then be displayed
4. Close the dropdown option menu by mouse clicking on the dropdown's "carrot down icon"
5. The panel containing the dropdown will scroll to the beginning of the horizontal scrollable  viewable region

To reproduce the keyboard problem
1. Run the reproducer: https://stackblitz.com/edit/nkd8th
2. Mouse Click on the dropdown (or navigate to the dropdown and press the enter key)
3. The dropdown option menu should then be displayed
4. Click the "down arrow key"
5. The panel containing the dropdown will scroll to the beginning of the horizontal scrollable  viewable region

### Expected behavior

The panel containing the dropdown should not scroll

**Comment by rosenthalj** [comment]
For completeness,  I searched the PrimeNG code baseline for  other places where there may be similar issues.  See annotated screenshot listed below.

![Code_search_results_and_Karma](https://github.com/primefaces/primeng/assets/45439491/0c7c2ece-f9a8-4b81-a14d-1d40e7e65383)

**Comment by cetincakiroglu** [comment]
Hi,

We've fixed two issues you've mentioned with the version 16.7.2, they both works as expected in the latest version.
Here is the example containing both issues happening in 16.7.1 https://stackblitz.com/edit/z78n8c?file=src%2Fapp%2Fdemo%2Fdropdown-basic-demo.html

Closing this and https://github.com/primefaces/primeng/issues/14078 since it's already fixed within 16.7.2. 

Thanks for reporting the issue!
